.env.dev

```diff
@@ -0,0 +1,17 @@
+# DB CONFIG
```

Comment:
Should we also provide `DGRAPH_GRPC_HOST` & `NESTJS_PORT` in this file ?

# NESTJS_CONFIG
NESTJS_PORT=...

# DGRAPH CONFIG
DGRAPH_GRPC_HOST=...

Reply:
I think `NESTJS_PORT` is needed but not the DGRAPH config values as I was able to set up the docker env without it.